lib/taskana-core/src/main/java/pro/taskana/report/api/row/TimestampRow.java

```diff
@@ -1,11 +1,12 @@
-package pro.taskana.impl.report.row;
```

Comment:
this should be internal

Reply:
TimestampRow is returned by TimestampReport.getRow(). Therefore, I would consider it part of the API - after all it is a datastructure that the client code must understand ...